src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1556,3 +1556,13 @@ def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     namespace.progress_hook = progress_callback
+
+
+def get_not_none_validator(attribute_name):
+    def validate_not_none(cmd, namespace):
+        attribute = getattr(namespace, attribute_name, None)
+        options_list = cmd.arguments[attribute_name].type.settings.get('options_list')
+        if attribute in (None, ''):
+            from azure.cli.core.azclierror import InvalidArgumentValueError
+            raise InvalidArgumentValueError('Argument {} should be specified'.format('/'.join(options_list)))
```

Comment:
this should be a `RequiredArgumentMissingError`, right?

Reply:
Actually they provide the required argument, but the value is None or '' which is invalid